### DISCUSSION TITLE
How to set `--ignore-last-opened` in config file?

### DISCUSSION THREAD

**Discussion by bingliu221** [coder/code-server#3056] [Q&A] [answered]
Config file is convenience, I wonder if there is a way to set `--ignore-last-opened` in config file?

**Comment by jsjoeio** [comment, marked as answer]
Yup! Any flags should be configurable in the config file. Since this is a boolean flag, you can set it in the config with:
```
ignore-last-opened: true
```

## Demo

Here's a demo:


https://user-images.githubusercontent.com/3806031/113906671-5823ff80-9789-11eb-98b7-4519d5b52f4c.mov

## Feedback

If you think this could be more clear/easier to find in the docs, PRs are welcome! ♥️

**Reply by bingliu221** [reply]
It's very helpful, thanks.
